src/Mapbender/CoreBundle/Resources/public/mapbender.element.legend.js

```diff
@@ -130,8 +130,14 @@
             } else if(context && options.changed && options.changed.children) {
                 for(layerName in options.changed.children) {
                     var layer = options.changed.children[layerName];
+
                     if(layer.state) {
+
                         if(layer.state.visibility) {
+
+                            if($('li.image[data-id="' + layerName + '"]', context).attr("data-gisurl"))
+                                $('li.image[data-id="' + layerName + '"]', context).html('<img src="' + $('li.image[data-id="' + layerName + '"]', context).attr("data-gisurl") + '">');
```

Comment:
We should not create HTML from strings, we should prefer creating it like @eSlider [have made it](https://github.com/mapbender/mapbender/blob/2971dc6c735e586da5ede1783ba963145067cb4d/src/Mapbender/CoreBundle/Resources/public/mapbender.element.legend.js#L334)

Reply:
So the if ($('li.image[data-id ... is the only functional change and everything else on this pull is just whitespace reformatting...?